setup-machine.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```

Comment:
what does this change do? I'm not used to seeing anything other than `#!`

Reply:
oh ... I don't know how this got in here or why I did this lol, will revert